samples/nfc/tag_reader/src/main.c

```diff
@@ -173,7 +204,13 @@ static void ndef_data_analyze(const u8_t *ndef_msg_buff, size_t nfc_data_len)
 		printk("Error during parsing a NDEF message, err: %d.\n", err);
 	}
 
-	nfc_ndef_msg_printout((struct nfc_ndef_msg_desc *) desc_buf);
+	ndef_msg_desc = (struct nfc_ndef_msg_desc *) desc_buf;
+
+	nfc_ndef_msg_printout(ndef_msg_desc);
+
+	for (u32_t i = 0; i < ndef_msg_desc->record_count; i++) {
```

Comment:
`u32_t` -> `size_t` Please use `size_t` to be consistent with the sample

Reply:
done